init-tools.cmd

```diff
@@ -0,0 +1,79 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+:CheckOS
+IF "%PROCESSOR_ARCHITECTURE%"=="x86" (set bit=x86) else (set bit=x64)
```

Comment:
1. that is a change in our current dev workflow.  I really liked to be able to just say "build -Architecture x86" and it just work from any of my command prompts.
2. Are we sure we can "have the right command prompt" on Jenkins and in VSO?

Let's not try to change this now.  Stick to what is currently working, and we can refactor later, if we feel necessary.

---
In reply to: [68865996](https://github.com/dotnet/cli/pull/3719#discussion_r68865996) [](ancestors = 68865996)